### PR TITLE
fix (structure) Fix structure of frontend

### PR DESCRIPTION
**Pull request by RomanMahola**
- Add .env.example to frontend directory
- Change configs

### CODE REVIEW

File: frontend/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VITE_API_ORIGIN_URL=http://localhost:3001/api/v1
+VITE_SENTRY_DSN='your sentry dsn'
+SENTRY_ENABLE=false
```
Comment (oleksandr-danylchenko) [inline frontend/.env.example]: `SENTRY_ENABLE` also needs to have the `VITE_` prefix. W/o it, the variable won't be recognized
https://vitejs.dev/guide/env-and-mode.html#env-files

Comment (oleksandr-danylchenko) [reply frontend/.env.example]: We also miss the `VITE_SENTRY_DSN`

File: frontend/src/configs/config.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+export interface ConfigInterface {
+  APP: AppConfig;
+}
+
+const configuration = (): ConfigInterface => {
+  config();
+
+  const { VITE_API_ORIGIN_URL, VITE_SENTRY_DSN, SENTRY_ENABLE } = process.env;
```
Comment (oleksandr-danylchenko) [inline frontend/src/configs/config.ts]: We **cannot** import env variables from the `process.env` in Vite. The variables must be imported from the `import.meta.env`

**Review by oleksandr-danylchenko — CHANGES_REQUESTED** [review]
Needs some Vite-specific polishing

File: frontend/src/configs/config.ts (hunk 2 above)
Comment (RomanMahola) [reply frontend/src/configs/config.ts]: Do we need additional config to import.meta.env. I mean path to .env file

Comment (oleksandr-danylchenko) [reply frontend/src/configs/config.ts]: No, if the env file is in the same directory as the `package.json`. Actually... we don't even need to use the `dotenv` explicitly by ourselves, bc `vite` uses it under the hood already

**Pull request merged**